lib/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "strict": false,
     "declaration": true,
     "declarationDir": "dist",
-    "emitDeclarationOnly": true
+    "emitDeclarationOnly": true,
+    "ignoreDeprecations": "5.0"
```

Comment:
Oh, so this was the fix to the broken CI? I'm curious to know what happened. Good catch <3